docs/introduction/framework-extensibility.md

```diff
@@ -0,0 +1,31 @@
+# Framework Extensibility
+
+This article summarizes current possibilities of the framework extension,
+provides a list of customizations that are not achievable now but are planned to be enabled soon, 
+as well as a list of customizations that are not (and will not be) possible at all.
+
+## What is achievable easily
```

Comment:
I would add here things that are easily achievable because you can do them in the open-box, eg. adding new entities, changing the FE design (and FE javascripts) etc.

As a new user of SSFW this could be extra helpful